docs/se/guides/dbclient.adoc

```diff
@@ -116,49 +115,49 @@ docker build -f Dockerfile.h2 . -t h2db
 docker run --rm -p 8082:8082 -p 9092:9092 --name=h2 h2db
 ----
 
-==== From Command Line
-
-A database stores the books from the library. H2 is a java SQL database easy to use and light.
-If H2 is not installed on your machine, here are few steps to quickly download and set it up:
+==== From the Command Line
 
-    * Download the latest H2 version from the official website: https://www.h2database.com/html/main.html
-    ** On windows, let guides yourself by the H2 installer.
-    ** On another platform, unzip the downloaded file into your wished directory.
-    ** Only the h2-\{latest-version}.jar, located into h2/bin folder, will be needed.
-    * Run the following command to start H2, do not forget to replace `\{latest-version}` by your current version.
+    1. Download the latest H2 version from the official website: https://www.h2database.com/html/main.html
+    
+    Note: Windows operating system users can download the Windows Installer.
+    
+    2. Unzip the downloaded file into your directory.
+    * Only the h2-\{latest-version}.jar, located in the h2/bin folder, will be needed.
+    
+    3. Open a terminal window and run the following command to start H2: 
+    
 
 [source,bash]
-.Open a terminal and enter the following:
+.Replace `\{latest-version}` with your current H2 version:
 ----
 java -jar h2-\{latest-version}.jar -webAllowOthers -tcpAllowOthers
 ----
 
-==== Run H2
-
-This output should be printed:
 
 [source,bash]
-.Terminal output
+.Print the terminal output
```

Comment:
This is the expected output of the command entered above. The verb "Print" seems to change the meaning (e.g. I would expect to be asked to print it).